### DISCUSSION TITLE
Which Editors would you like Git to use: Emacs?

### DISCUSSION THREAD

**Discussion by dieter-wilhelm** [git-for-windows/git#3702] [Q&A] [answered]
Is Emacs not considered a worthy text editor among the list of offered alternatives from the installer?

      - Gnu Nano
      - Vim (the ubiquitous text editor) [default] 
      - Notepad++
      - Visual Studio Code
      - Visual Studio Code Insiders
      - Sublime Text
      - Atom
      - VSCodium
      - Notepad
      - Wordpad
 
Please tell me (your) reasons for this, thank you

**Comment by dscho** [comment, marked as answer]
The reason is that nobody put in the effort yet to support that option. You could be the one who makes it happen.

**Comment by dieter-wilhelm** [comment]
Thank you for the explanation.
I think such support is exclusively implemented in this file: https://github.com/git-for-windows/build-extra/blob/main/installer/install.iss, isn't it?

By the way, I tried to use emacsclient with a command line option. But it seems that the installer is only accepting  executables without options.  I also tried to use quotes in various ways (", ') but to no avail.  Is there a special way to quote command line options?  The installer mostly complains that a thing like "c:\Program " is not an executable...

Thanks again

**Reply by dscho** [reply]
Yes,`install.iss` is the place.

VS Code gets called with `-w`, so there is precedent.

The value is actually a command-line, therefore you need to quote everything properly. I _think_ there's `'+#34+'` type of double-quoting in `install.iss`...
